src/pages/digital-samhandling-offentlig-privat/detaljer/DsopDetaljer.tsx

```diff
@@ -10,13 +10,17 @@ interface Routes {
 }
 
 const DsopDetaljer = () => {
-  const params = useParams<Routes>();
+  const params = useParams<Readonly<Params<keyof Routes>>>();
   const { id } = params;
 
   useEffect(() => {
     window.scrollTo(0, 0);
   }, []);
 
+  if (typeof id === "undefined") {
```

Comment:
Ser at dette er løst på to forskjellige måter her og i DetaljertArbeidsforhold, hvis ikke jeg har misforstått noe. Tror jeg foretrekker denne løsningen personlig.

Reply:
Enig! Fikser i DetaljertArbeidsforhold!